var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -583,6 +583,8 @@ class Llvm(CMakePackage):
 
     # Github issue #4986
     patch('llvm_gcc7.patch', when='@4.0.0:4.0.1+lldb %gcc@7.0:')
+    # Backport from llvm master and add additional test.
+    patch('llvm_gcc6.patch', when='@7.0.1:8.0.0+lldb %gcc@6:')
```

Comment:
Remove the gcc version restriction since it's not checked in upstream.  I also think the spec can be generalized even more since it seems to not be related to `+lldb` and is pertinent for at least the entire 7.x and 8.x branches based on looking at the llvm source tree.  So would this be more appropriate?
```
patch('llvm_gcc_ppc.patch', when='@7:8 %gcc')
```

Reply:
Sounds good.